api/handlers.go

```diff
@@ -186,3 +246,21 @@ func checkLimitOffset(l *server.LimitParam, o *server.OffsetParam) (int, int, er
 
 	return limit, offset, nil
 }
+
+func checkJetDropID(jetDropID *server.FromJetDropId) (*string, error) {
+	if jetDropID == nil {
+		return nil, nil
+	}
+	str := string(*jetDropID)
+	s := strings.Split(str, ":")
+	if len(s) != 2 {
+		return nil, fmt.Errorf("wrong jet drop id format")
```

Comment:
Why  use`Errorf` if we don't do formatting?

Reply:
fixed